package.json

```diff
@@ -1,36 +1,28 @@
 {
   "name": "frontend",
   "version": "0.1.0",
+  "proxy": "http://localhost:8000",
   "private": true,
   "homepage": ".",
-  "proxy": "http://localhost:8000",
   "dependencies": {
     "@emotion/react": "^11.7.1",
     "@emotion/styled": "^11.6.0",
     "@mui/icons-material": "^5.2.5",
     "@mui/material": "^5.2.6",
+    "@react-google-maps/api": "^2.7.0",
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
+    "http-proxy-middleware": "^2.0.3",
+    "npm": "^8.5.1",
```

Comment:
I don't think npm is a project dependency?

Reply:
This was not added by me, maybe when I was verifying that the npm and node were up to date it was added. I removed it.